### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by cbsaikumar**
Please review Homework 6 and let me know incase of any comments. Thank you :)

### CODE REVIEW

**Review by abruzgin — COMMENTED** [review]
Hey, I can't find any place where you launch your DB in project. You should first call `.sync()` to launch the DB and after that launch the project

**Comment by cbsaikumar** [comment]
> Hey, I can't find any place where you launch your DB in project. You should first call `.sync()` to launch the DB and after that launch the project

**Comment by cbsaikumar** [comment]
Fixed the issues as per the review comments. I was not using Sequelize-cli to automatically create the tables from the models earlier. I fixed it now. It works as expected now. Thanks and appreciate the help.

File: src/expressjs/controllers/products.js [hunk 1]
```diff
@@ -1,33 +1,44 @@
...
 
 export const getProductReviews = (req, res) => {
     const { id } = req.params;
 
-    const reviews = products.default.find(product => product.id === parseInt(id, 10)).reviews;
-    
-    return res.json(reviews);
+    const reviews = Product.findOne({
+        where: {
+            id: id
```
Comment (abruzgin) [inline src/expressjs/controllers/products.js]: please use ES6 shorthand object properties

File: src/expressjs/models/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.json')[env];
+const db = {};
+
+let sequelize;
```
Comment (abruzgin) [inline src/expressjs/models/index.js]: If you dont use further reassignmsents of `sequelize`, use ternary operator instead
```javascript
const sequelize = config.use_env_variable
  ? new Sequelize(process.env[config.use_env_variable], config)
  : new Sequelize(config.database, config.username, config.password, config);
```

**Comment by cbsaikumar** [comment]
Hi... I closed it by mistake. Sorry for that. Please review now :)

File: src/expressjs/controllers/products.js (hunk 1 above)
Comment (cbsaikumar) [reply src/expressjs/controllers/products.js]: Fixed it now thanks

File: src/expressjs/models/index.js (hunk 2 above)
Comment (cbsaikumar) [reply src/expressjs/models/index.js]: Fixed it now thanks

**Pull request merged**